### PR TITLE
West Midlands Class 3 - Selahaddin Alan - Git & GitHub - Week 2

### PR DESCRIPTION
**Pull request by justsealan**
West Midlands Class 3 - Selahaddin Alan - Git & GitHub - Week 2

# Your Details

- Your Name: Selahaddin Alan
- Your City: Nottingham
- Your Slack Name: Selahaddin Alan

# Homework Details

- Module: Git & GitHub
- Week: 2

-----
[View rendered homework.md](https://github.com/justsealan/GitHomeworkFixErrors/blob/main/homework.md)

### CODE REVIEW

File: homework.md [hunk 1]
```diff
@@ -92,13 +93,13 @@ A place for us to store and share our code.
 
 What does it mean to "Commit your changes"?
 
-Send you code to Github
+Send you code to Github.
```
Comment (Halla-S) [inline homework.md:97]: Hi ,You might have missed to correct this one

**Review by Halla-S — COMMENTED** [review]
Well done

File: homework.md [hunk 2]
```diff
@@ -92,13 +93,13 @@ A place for us to store and share our code.
 
 What does it mean to "Commit your changes"?
 
-Send you code to Github
+Save the changes you made.
```
Comment (gynaa78) [inline homework.md:97]: Yes, to save changes either to a local repo or to a remote github web.

**Review by gynaa78 — APPROVED** [review]
Well done Selahaddin!

File: homework.md (hunk 1 above)
Comment (Halla-S) [inline homework.md:97]: To save your changes

**Pull request closed without merging**